aten/src/ATen/cpu/vec/vec256/vec256_int.h

```diff
@@ -680,6 +680,11 @@ class Vectorized8 : public Vectorizedi {
   static Vectorized<T> loadu(const void* ptr) {
     return _mm256_loadu_si256(reinterpret_cast<const __m256i*>(ptr));
   }
+  static Vectorized<T> loadu_one_fourth(const void* ptr) {
+      // Fast path if only load element number of 8
+      __m128i input_128 = _mm_loadl_epi64(reinterpret_cast<const __m128i*>(ptr));
+      return _mm256_castsi128_si256(input_128);
+  }
   static Vectorized<T> loadu(const void* ptr, T count) {
```

Comment:
can we just add this `loadu_one_fourth` as a fast path of `loadu(ptr, count)`?

Reply:
As we synced offline, add a note for why we add this fast path.